conf/modules.config

```
@@ -883,4 +883,28 @@ process {
             enabled: true
         ]
     }
+
+    //
+    // SEXDETERRMINE
+    //
+    withName: SAMTOOLS_DEPTH {
+        tag = { "${meta1.reference}|${meta1.sample_id}_${meta1.library_id}" }
+        ext.prefix = { "${meta1.sample_id}_${meta1.library_id}_samtoolsdepth" }
+        publishDir = [
+            path: { "${params.outdir}/sex_determine/samtools_depth/" },
+            mode: params.publish_dir_mode,
+            enabled: true
```

```suggestion
            enabled: false
```
Don't need to output this imo.